docker/docker-compose.test.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.5'
+services:
+    blockscout-tests-db:
+        container_name: blockscout-tests-db
+        image: postgres
+        environment:
+            - POSTGRES_USER=postgres
+            - POSTGRES_PASSWORD=postgres
+            - POSTGRES_DB=explorer_test
+    blockscout-tests-runner:
+        container_name: blockscout-tests-runner
+        depends_on:
+            - blockscout-tests-db
+        image: hermez-node-blockscout:latest
+        environment:
+            - NETWORK=POE
+            - SUBNETWORK=Polygon Hermez
+            - ECTO_USE_SSL=false
+            - LOGO=/images/blockscout_logo.svg
+            - LOGO_FOOTER=/images/blockscout_logo.svg
+            - PGHOST=blockscout-tests-db
+            - PGPORT=5432
+            - PGDATABASE=explorer_test
```

Comment:
We are passing down this variable from here but it's still hardcoded in the other file. Same question as before, do we need to send the same value as the default one from here?

Reply:
Good catch! I added the 5 env vars described in [the docs](https://www.postgresql.org/docs/9.3/libpq-envars.html) but forgot to read this one. Added. Now the database can also be configured.